guides/theming.md

```diff
@@ -367,6 +367,83 @@ $my-palette: mat.define-palette(mat.$indigo-palette);
 }
 ```
 
+## Strong focus indicators
+
+By default, most components indicate browser focus by changing their background color as prescribed
+by the Material Design specification. This behavior, however, can fall short of accessibility
+requirements, such as [WCAG][], which require a stronger indication of browser focus.
+
+Angular Material supports rendering highly visible outlines on focused elements. Applications can
+enable these strong focus indicators via two Sass mixins:
+`strong-focus-indicators` and `strong-focus-indicators-theme`.
+
+The `strong-focus-indicators` mixin emits structal indicator styles for all components. This mixin
```

Comment:
Is "structal" supposed to be "structural" or "strong"?

Reply:
I did intent to say "structural" versus the color styles from `strong-focus-indicators-theme`. Would it be better to say "base" or something similar here?

Reply:
Structural makes sense, thanks!